library/Zend/Mail/Part.php

```diff
@@ -47,7 +47,7 @@
  * @copyright  Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
  * @license    http://framework.zend.com/license/new-bsd     New BSD License
  */
-class Zend_Mail_Part implements RecursiveIterator, Zend_Mail_Part_Interface
+class Zend_Mail_Part implements RecursiveIterator, Zend_Mail_Part_Interface, Countable
```

Comment:
The tests run `count` on this object, so added the `Countable` interface. It doesn't seem to happen in the code base at all otherwise.

The specific test doing this was counting the object to make sure there were no parts.